soc/nxp/kinetis/ke1xz/Kconfig

```diff
@@ -12,3 +12,5 @@ config SOC_SERIES_KE1XZ
 	select CLOCK_CONTROL
 	select HAS_MCUX
 	select PLATFORM_SPECIFIC_INIT
+	select HAS_MCUX_LPSPI
+	select HAS_MCUX_EDMA
```

Comment:
please remove these, they arent needed

Reply:
They are used to test `tests/drivers/spi/spi_loopback` to avoid being filtered out.
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/spi/spi_loopback/testcase.yaml
![image](https://github.com/user-attachments/assets/aeed18d2-d154-45a5-a992-581efbb9617d)

Reply:
please change it to use the DT_HAS macros then (for example see the flexio testcase in this file also using the DT_HAS macro)

Reply:
Updated